### DISCUSSION TITLE
On-chain Transaction Stuck

### DISCUSSION THREAD

**Discussion by roarkjanis-resale** [Ride-The-Lightning/RTL#1282] [Q&A] [answered]
About five weeks ago I created an on-chain transaction in RTL to send some coins back to my regular wallet. I made the mistake of setting the fee to low and it languished for weeks in the mempool. I figured it would eventually get purged and the coins would return to my on-chain balance in RTL and I could try again. The transaction is gone from the mempool but is still showing as pending in RTL. Is there a way to cancel it in RTL so the coins will show as available balance again?

**Comment by saubyk** [comment, marked as answer]
Hi, have you tried restarting your node? Also which implementation are you on?
Depending on the implementation, you can also try a wallet rescan.

**Reply by roarkjanis-resale** [reply]
Thank you for suggestions. I have done a restart of my node. I'm using the implementation built into myNode. I've looked all over and cannot find any option to do a wallet rescan. Any other ideas based on using the myNode implementation?

**Reply by saubyk** [reply]
So, I am assuming yours is an LND node. You can confirm this by looking at the `Node Information` panel on the RTL Dashboard. Below are instructions on what you need to do if you are running a LND node.

Wallet rescanning is an advanced operation. You may need to access your device via the command line and you'd need to restart your LND node with a specific setting in the `LND.conf` file.
Set this parameter in the conf file: `reset-wallet-transactions=true`, before restarting the node

Here's the description of what it does from the [conf file documentation](https://github.com/lightningnetwork/lnd/blob/master/sample-lnd.conf)
```
Removes all transaction history from the on-chain wallet on startup, forcing a
full chain rescan starting at the wallet's birthday. Implements the same
functionality as btcwallet's dropwtxmgr command. Should be set to false after
successful execution to avoid rescanning on every restart of lnd.
```
Once you've found the UTXOs, you should remove this setting from your conf file and restart your node again.

Hope this helps. And please stop making 1 sat/vbyte transactions, or check mempool for prevailing fee rates before doing onchain operations.

**Reply by roarkjanis-resale** [reply]
Thank you for the instructions. This should fix the problem. I will find the file, make the changes, etc.

As info, I did check the mempool before making the transaction and it was showing 4 sat/vbyte for no priority and since I wasn't in a hurry I used that. I probably show have used something higher as I didn't have a good sense of how the 4 really never gets confirmed. Since them I've watched the mempool a lot and learned. Thank you for the advice.

**Reply by roarkjanis-resale** [reply]
Following up to let you know this solution worked. As info, it took me a bit to figure it out and myNode support helped but you cannot edit the LND.conf file directly via terminal and have to via their web interface. Here is the link to the help for this in case somebody else comes across this problem.

https://mynodebtc.github.io/advanced/customize-config.html#customizing-the-your-lnd-configuration

Thank you for all your help.
